internal/process/process_posix.go

```diff
@@ -0,0 +1,28 @@
+// +build !windows
+
+package process
+
+import (
+	"os/exec"
+	"syscall"
+	"time"
+)
+
+func gracefulStop(cmd *exec.Cmd, timeout time.Duration) {
+	time.AfterFunc(timeout, func() {
+		if cmd == nil || cmd.ProcessState == nil {
```

Comment:
I guess you're right about that, but I did have some panics that lead to me adding this. It could be that the panic was only around cmd.ProcessState, but I don't recall.